ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -269,14 +298,54 @@ public boolean updateBlobExpiration(BlobId blobId, long expirationTime) throws C
     long now = System.currentTimeMillis();
     long retentionThreshold = now - retentionPeriodMs;
     SqlQuerySpec deadBlobsQuery = new SqlQuerySpec(DEAD_BLOBS_QUERY_TEMPLATE,
-        new SqlParameterCollection(new SqlParameter(THRESHOLD_PARAM, retentionThreshold)));
+        new SqlParameterCollection(new SqlParameter(LIMIT_PARAM, deadBlobsQueryLimit),
+            new SqlParameter(THRESHOLD_PARAM, retentionThreshold)));
     try {
       return cosmosDataAccessor.queryMetadata(partitionPath, deadBlobsQuery, azureMetrics.deadBlobsQueryTime);
     } catch (DocumentClientException dex) {
       throw new CloudStorageException("Failed to query dead blobs for partition " + partitionPath, dex);
     }
   }
 
+  @Override
+  public List<CloudBlobMetadata> findEntriesSince(String partitionPath, CloudFindToken findToken,
+      long maxTotalSizeOfEntries) throws CloudStorageException {
+    SqlQuerySpec entriesSinceQuery = new SqlQuerySpec(ENTRIES_SINCE_QUERY_TEMPLATE,
+        new SqlParameterCollection(new SqlParameter(LIMIT_PARAM, 1000),
```

Comment:
Make the `1000` a static final variable?

Reply:
sure